roles/sap_vm_provision/tasks/common/set_ansible_vars.yml

```diff
@@ -39,7 +38,7 @@
     sap_hana_install_instance_nr: "{{ sap_hana_install_instance_nr | default(sap_system_hana_db_instance_nr) }}"
     sap_hana_install_use_master_password: "{{ sap_hana_install_use_master_password | default('y') }}"
     sap_hana_install_master_password: "{{ sap_hana_install_master_password | default('') }}"
-    sap_hana_install_software_directory: "{{ sap_software_download_directory | default('/software') }}"
+    sap_hana_install_software_directory: "{{ sap_install_media_detect_source_directory | default('/software') }}"
```

Comment:
This change is not needed since hana_install role does not handle `sap_install_media_detect_source_directory` by itself, so lets keep it handled by AP4S variable `sap_software_download_directory `

`sap_install_media_detect_source_directory` is anyway handled through default task above it.

Reply:
Good point, I'll remove `sap_hana_install_software_directory`